styles.css

```diff
@@ -0,0 +1,78 @@
+body {
+  font-family: 'Poppins', sans-serif;
+  margin: 0;
+}
+
+main {
+  background: #f7f7f9;
+  padding-top: 75px;
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+.header {
+  position: fixed;
+  width: 90vw;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 30px 24px;
+  background-color: #fff;
+  box-shadow: 0 2px 6px 0 rgb(0 0 0 / 5%), 0 2px 15px 0 rgb(0 0 0 / 15%);
+}
```

Comment:
- [ ] Kindly make sure your header section and the menu icon look as close as possible to the Figma design 🙏 
- [ ] Kindly remove the `box shadow` as you won't be needing it for this design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/185520482-b5d2f9e6-525c-42d7-aaff-3dad8776edbc.png)  |  ![image](https://user-images.githubusercontent.com/74814780/185520468-70b5b4aa-3797-47b9-80fb-50f6636867c3.png)